bach/bach/dataframe.py

```diff
@@ -2439,6 +2434,68 @@ def drop_duplicates(
         self._update_self_from_df(df)
         return None
 
+    def dropna(
+        self,
```

Comment:
```suggestion
        self,
        *,
```
this forces our users to explicitly write `dropna(axis=1)` instead of `dropna(1)`. The latter will give a warning with Pandas about that usage being deprecated. I think we don't need the warning and can just directly force the 'right' behaviour. Will save us a lot of deprecation messages if we want to remain compatible in the future